tools/tck-integrity-tests/src/main/scala/org/opencypher/tools/tck/FeatureFormatChecker.scala

```diff
@@ -39,10 +39,18 @@ import scala.util.Try
 
 class FeatureFormatChecker extends TCKCucumberTemplate {
 
+  private var currentScenarioName = ""
+  private var withIntentionalStyleViolation: Boolean = false
+
+  Before { (scenario: cucumber.api.Scenario) =>
+    validateDuplicateNames(scenario).map(msg => throw InvalidFeatureFormatException(msg))
+    currentScenarioName = scenario.getName
+    withIntentionalStyleViolation = scenario.getSourceTagNames.contains("@withIntentionalStyleViolation")
```

Comment:
I would move `@withIntentionalStyleViolation` to a constant. Perhaps in `TCKConstants` class?